### DISCUSSION TITLE
vscode로 디버그하고 실행할수 있게 되었습니다.

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#170] [General]
왜 갑자기 되는진 모르겠습니다.

![image](https://user-images.githubusercontent.com/62102830/211849298-27c2aa06-dad9-4cc1-a6ad-0f141e7e3857.png)

**Comment by Woon-2** [comment]
부럽군요... 저도 vscode나 visual studio 쓰고 싶습니다.

**Reply by Lcat12345** [reply]
하루 빨리 시간이 가길..
